README.md

```diff
@@ -32,4 +32,27 @@ Aby odpalić projekt należy posiadać:
 ```shell
     $ npm run test
 ```
+---
+### Uruchomienie projektu z wykorzystaniem: watch & nodemon
+- Przejdz do katalogu projektu i wykonaj następujące polecenia: 
+   
+```shell
+   $ npm i
+``` 
+- Uruchomienie: `servera`, `clienta` z wykorzystaniem: `watchera` oraz `nodemon`
+ 
+```shell
+   $ npm run watch
+```
+Ta komenda pozwala na uruchamienie skryptów jakie zostały dodane do pliku: `package.json`.
+
+#### Zmianty jakie zostały dodane do: `package.json`
+
+```JS
+    "watch": "run-p watch:server watch:client",
+    "watch:server": "nodemon -w src/server/ src/server/app.js",
+    "watch:client": "webpack --watch",
+``` 
+
+
```

Comment:
mega fajnie ze pomyslales o readme, zmian nie opisuj bo to mozna zobaczyc w pliku, napisz tylko ze do codziennej pracy odpalamy npm run watch

Reply:
Porawione